core/capabilities/remote/types/message.proto

```diff
@@ -19,11 +19,27 @@ message MessageBody {
   uint32 version = 1;
   bytes sender = 2;
   bytes receiver = 3;
-  bytes message_id = 4; // scoped to (don_id, capability_id)
-  string capability_id = 5;
-  string don_id = 6; // where the capability actually lives
-  string method = 7;
-  int64 timestamp = 8;
-  bytes payload = 9;
+  int64 timestamp = 4;
+  bytes message_id = 5; // scoped to sender
+  string capability_id = 6;
+  string capability_don_id = 7;
+  string caller_don_id = 8;
+  string method = 9;
   Error error = 10;
+
+  // payload contains a CapabilityRequest or CapabilityResponse
+  bytes payload = 11;
```

Comment:
@bolekk nit/follow up: can we make this the actual corresponding proto?

Reply:
hmm I'm not sure if we want to do that but let me think about it.